test/passing/tag_only.mli

```diff
@@ -1,64 +1,80 @@
-(** @deprecated  *)
 open Module
+(** @deprecated  *)
 
+open Module
 (** abc
     @deprecated  *)
-open Module
 
 (** @inline *)
 include sig
   type t
 end
 
-(** @inline *)
 include Type
-
 (** @inline *)
+
 include module type of Module
+(** @inline *)
 
-(** @deprecated  *)
 module A : B
+(** @deprecated  *)
 
 (** @deprecated  *)
 module A : sig
   type t
 end
 
-(** @open *)
 module type A = B
+(** @open *)
 
 (** @open *)
 module type A = sig
   type t
 end
 
+(** @deprecated  *)
+type t = T
+
+type t = t
+(** @deprecated  *)
+
+val a : b
+(** @deprecated  *)
+
 [@@@ocamlformat "doc-comments-tag-only=fit"]
 
-open Module (** @deprecated  *)
+open Module  (** @deprecated  *)
 
+open Module
 (** abc
     @deprecated  *)
-open Module
 
 (** @inline *)
 include sig
   type t
 end
 
-include Type (** @inline *)
+include Type  (** @inline *)
 
-include module type of Module (** @inline *)
+include module type of Module  (** @inline *)
 
-module A : B (** @deprecated  *)
+module A : B  (** @deprecated  *)
 
 (** @deprecated  *)
 module A : sig
   type t
 end
 
-module type A = B (** @open *)
+module type A = B  (** @open *)
```

Comment:
If this intended?

Reply:
Yes. This is the `doc_comments_padding` option. It was only used on type declarations until now.
@jberdine What do you think of this behavior ?